### ISSUE TITLE
LibreSpeed Redesign

### ISSUE THREAD

**Issue by Chris-ZoGo** [librespeed/speedtest#585]
## Description
During Hacktoberfest 2022 & 2023, [fromScratch](fromscratch.io) Studio took on the request for LibreSpeed redesign and UI improvements.
We ran 2 design sprints one on '22 and one on '23, and produced high-fidelity screens for LibreSpeed redesign. 
Furthermore, this year, we produced high-fidelity screens for mobile-view as well. 

## Why it should be implemented
Implementing a fresh look on LibreSpeed will attract more users, and improve the usability and experience.

## Optional: screenshots
**Screenshots of what was designed**

**Desktop**
<img width="1177" alt="Screenshot 2023-11-10 at 3 22 41 PM" src="https://github.com/librespeed/speedtest/assets/69966222/50bb1d0b-f196-4e39-b09d-6585ac1d67fb">

**Mobile**
<img width="372" alt="Screenshot 2023-11-10 at 3 24 50 PM" src="https://github.com/librespeed/speedtest/assets/69966222/32025fdc-704d-4493-a03b-b71a192466b0">

Figma link
(feel free to look around)
https://www.figma.com/file/rAfMAZSy56eKHdoh7FrxRL/LibreSpeed-Redesign?type=design&node-id=373%3A501&mode=design&t=kjLTOvnD6syP7coj-1

**Comment by Chris-ZoGo** [comment]
@sstidl Hello Stefan! 
We would appreciate any feedback/thoughts on this.
And if you decide to approve it we could create separate (sub?) issues for all the necessary components/elements and screens.

**Comment by Chris-ZoGo** [comment]
Here is a public view link!
https://www.figma.com/file/rAfMAZSy56eKHdoh7FrxRL/LibreSpeed-Redesign?type=design&node-id=0%3A1&mode=design&t=196EX9PiBDxmw9PZ-1

**Comment by sstidl** [comment]
@gary-host-laptop before we can merge it, someone has to implement the change and open a pull request for it.
If you are able to to it, feel free to take the task

**Comment by Chris-ZoGo** [comment]
If there is anything you need from our side (the design team) please let me know :)

**Comment by sstidl** [comment]
> If there is anything you need from our side (the design team) please let me know :)

Could you provide stylesheet file and a demo html page? Or how do I get those out of figma?

**Comment by Chris-ZoGo** [comment]
@sstidl I think that for something like that ( a stylesheet or a demo) we need a front-end developer to take over this task. 
It's not possible for us (as designers) to provide what you suggested. 
As far as handing off the design for development, this is what we usually do with other dev-teams. 
There is a dev-mode view in Figma, where developers can hover above the components and see all the details, even some code fragments. 

Maybe you can open a new issue asking for someone to extract/provide you with a stylesheet or demo page?

**Comment by Timendus** [comment]
>we need a front-end developer to take over this task

Done :wink:

**Comment by sstidl** [comment]
#649 is the PR

**Comment by Chris-ZoGo** [comment]
@Timendus It looks amazing! Thank you for trusting the design and for your effort and time :)

**Comment by sstidl** [comment]
Release 6.0.0 with the new design coming soon

**Comment by Yolakalemowa** [comment]
terrible design :( 
It's sad cause I used to love the extremely basic minimal to the point NO BULLSHIT design. can u provide users option to have that basic one?

**Comment by sstidl** [comment]
Still there on self hosted
